rules-threat-hunting/windows/process_creation/proc_creation_win_net_quic.yml

```diff
@@ -0,0 +1,28 @@
+title: SMB over QUIC Via Net.EXE
+id: 2238d337-42fb-4971-9a68-63570f2aede4
+related:
+    - id: 6df07c3b-8456-4f8b-87bb-fe31ec964cae
+      type: similar
+status: experimental
+description: Detects mount windows SMB over QUIC
```

Comment:
Detects the mounting of Windows SMB over QUIC which is not expected in an enterprise environment